addons/model-training/metadata.yaml

```diff
@@ -0,0 +1,15 @@
+name: model-training
+version: 1.0.0
+description: Enable training for models
+
+tags:
+- ai
+- training
+- alpha
+
+dependencies:
+  - name: fluxcd
+
+deployTo:
+  control_plane: true
+  runtime_cluster: true
```

Comment:
Please use `runtimeCluster`. And control plan is enable by default,  `control_plane` field is needn't.